packages/crayons-icon/gulpfile.js

```diff
@@ -33,6 +33,10 @@ const generateIconsExportData = async () => {
   };
   try {
     let indexData = '/* eslint-disable */';
+    indexData =
+      indexData +
+      '\n' +
+      "export const CRAYONS_ICONS_ASSET_PATH = 'https://unpkg.com/@freshworks/crayons-icon@canary/icons';";
```

Comment:
So based on this configuration we will need to update this URL to `crayons-icon@canary` => `crayons-icon@next` when we go to production? 

Is there a better way to do this?

Reply:
Need to think on it. Published url is not available untill we publish. However users can rgister crayons url from outside as well. But once we publish , we need a PR to update it.